test/e2e/create_test.go

```diff
@@ -53,17 +53,50 @@ var _ = Describe("Podman create", func() {
 	})
 
 	It("podman create container based on a remote image", func() {
-		session := podmanTest.Podman([]string{"create", BB_GLIBC, "ls"})
+		// Changing session -> create to avoid conflict for 2 session variable
+		create := podmanTest.Podman([]string{"create", BB_GLIBC, "ls"})
+		create.WaitWithDefaultTimeout()
+		Expect(create).Should(Exit(0))
+		Expect(podmanTest.NumberOfContainers()).To(Equal(1))
+
+		// Registry on 5000
+		lock := GetPortLock("5000")
+		defer lock.Unlock()
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "registry", "-p", "5000:5000", registry, "/entrypoint.sh", "/etc/docker/registry/config.yml"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-		Expect(podmanTest.NumberOfContainers()).To(Equal(1))
+
+		if !WaitContainerReady(podmanTest, "registry", "listening on", 20, 1) {
+			Skip("Cannot start docker registry.")
+		}
+		// Adding for create
+		create1 := podmanTest.Podman([]string{"create", "--tls-verify=false", ALPINE, "localhost:5000/my-alpine"})
+		create1.WaitWithDefaultTimeout()
+		Expect(create1).Should(Exit(0))
+		Expect(podmanTest.NumberOfContainers()).To(Equal(2))
+
```

Comment:
I think we can leave this test as is and just use the one below. It's effectively executing the same thing twice, so we can spare some test cycles.

Reply:
Yeah can be removed this one. 
Removed under podman create container `It("podman create using existing name", func() {} )`

Reply:
nit: extra line here

Reply:
Done!